Flashcards.DouglasFir/Flashcards/App1.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
What's up with this file? It doesn't seem to be used and in the `.csproj` it gets removed.

Reply:
Not sure how this snuck in there. This was removed early in development. Corrected and verified to be removed.